src/main/kotlin/no/nav/familie/ef/søknad/api/dto/pdl/MedForelder.kt

```diff
@@ -0,0 +1,51 @@
+package no.nav.familie.ef.søknad.api.dto.pdl
+
+import no.nav.familie.kontrakter.ef.søknad.Fødselsnummer
+import java.time.LocalDate
+import java.time.Period
+
+class MedForelder(_navn: String, val harAdressesperre: Boolean, val død: Boolean = false, val ident: String) {
+
+    val navn: String
+    val alder: Int
+
+    init {
+        val fødselsnummer = Fødselsnummer(ident)
+        this.alder = Period.between(fødselsnummer.fødselsdato, LocalDate.now()).years
+        this.navn = when (harAdressesperre) {
+            true -> ""
+            false -> _navn
+        }
+
+    }
+
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as MedForelder
+
+        if (harAdressesperre != other.harAdressesperre) return false
+        if (død != other.død) return false
+        if (ident != other.ident) return false
+        if (navn != other.navn) return false
+        if (alder != other.alder) return false
+
+        return true
+    }
+
+    override fun hashCode(): Int {
+        var result = harAdressesperre.hashCode()
+        result = 31 * result + død.hashCode()
+        result = 31 * result + ident.hashCode()
+        result = 31 * result + navn.hashCode()
+        result = 31 * result + alder
+        return result
+    }
```

Comment:
Tror ikke du trenger disse overridene. Om hashcode og equals grenereres på grunnlag av navn eller _navn, burde ikke medføre noen forskjell på resulatet av equals og hash.

Reply:
Disse testene feiler uten: (hash brukes for å vite om vi kan bruke mellomlagret søknad i ui)
`@Test
    fun `Lik søkerInfo med adressesperre skal ha lik hash`() {
        mockPdlHentBarn(adressebeskyttelseGradering = FORTROLIG)
        mockHentPersonPdlClient(fornavn = "Et navn", adressebeskyttelseGradering = FORTROLIG)
        val søkerinfo = oppslagServiceService.hentSøkerinfo()
        mockHentPersonPdlClient(fornavn = "Et navn", adressebeskyttelseGradering = FORTROLIG)
        val søkerinfo2 = oppslagServiceService.hentSøkerinfo()
        assertEquals(søkerinfo.hash, søkerinfo2.hash)
    }`

Reply:
Tenker du kan erstatte med `@JsonIgnore`. I tillegg burde man ha en integrasjonstest for dette som validerer json som returneres fra tjenesten som sjekker att _navn ikke er med 
```
data class Medforelder(@JsonIgnore private val _navn: String, val harAdressesperre: Boolean, val død: Boolean = false, val ident: String) {

    val navn: String
    val alder: Int

    init {
        val fødselsnummer = Fødselsnummer(ident)
        this.alder = Period.between(fødselsnummer.fødselsdato, LocalDate.now()).years
        this.navn = when (harAdressesperre) {
            true -> ""
            false -> _navn
        }
    }

    override fun toString(): String {
        return "MedForelder(harAdressesperre=$harAdressesperre, død=$død, alder=$alder)"
    }
}
```

Reply:
Endret tilbake til data-class. Fungerer bra. Trenger ikke @JsonIgnore så lenge variablen er private :) 
la til en integrasjonstest som jeg tror dekker dette.